keyboards/doodboard/duckboard_r2/duckboard_r2.c

```diff
@@ -15,3 +15,25 @@
  */
 
 #include "duckboard_r2.h"
+
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
Please put this under the vial keymap (e.g. keymap.c) to prevent conflicts with qmk merges